src/components/Counter.js

```diff
@@ -20,27 +19,27 @@ class Counter extends Component {
         return (
             <p>
                 Clicked: {this.props.count} times
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.increment() }>
                     +
                 </button>
-                <button onClick={() => /* Fill me in */ }>
+                <button onClick={() => this.props.decrement() }>
```

Comment:
Why not just use the functions themselves instead of putting them in the body of another?
For example:
```javascript
<button onClick={this.props.decrement}> 
```